Runtime/Extensions.cs

```diff
@@ -41,5 +42,22 @@ public static class Extensions
         {
             return IsDefined(serializableInterface, out value);
         }
+
+        /// <summary>
+        /// Convert a IEnumerable of Interfaces to a List of SerializableInterfaces
+        /// </summary>
+        public static List<SerializableInterface<T>> ToSerializableInterfaceList<T>(this IEnumerable<T> list) where T : class
+        {
+            return list.Select(e => new SerializableInterface<T>(e) ).ToList();
+        }
+
+         /// <summary>
+        /// Convert a IEnumerable of Interfaces to an Array of SerializableInterfaces
+        /// </summary>
+        public static SerializableInterface<T>[] ToSerializableInterfaceArray<T>(this IEnumerable<T> list) where T : class
+        {
+            return list.Select(e => new SerializableInterface<T>(e)).ToArray();
+        }
+
```

Comment:
```suggestion
        }
```

Reply:
I cant accept this Suggestion, Github says "This suggestion is invalid because no changes were made to the code"....